src/skills-builder/skills-builder-steps/view-results/test/ViewResults.test.jsx

```diff
@@ -54,7 +54,7 @@ describe('view-results', () => {
       renderSkillsBuilderWrapper();
       // Click the next button to trigger "fetching" the data
       await act(async () => {
-        fireEvent.click(screen.getByRole('button', { name: 'Next Step' }));
+        fireEvent.click(screen.getByRole('button', { name: 'Show Recommendations' }));
```

Comment:
I will usually pull the string I'm looking for from the messages so I don't have to change the tests if the text changes. No biggie, but should we trend toward that in the future?

Reply:
This is a great point! I went ahead and did that for all of the tests as it was pretty trivial to setup a `messages` object in the `setupSkillsBuilder` file. This way all messages are imported in one spot and can be accessed from any test.